commands/src/main/java/org/wildfly/extras/creaper/commands/ra/AddResourceAdapter.java

```diff
@@ -0,0 +1,82 @@
+package org.wildfly.extras.creaper.commands.ra;
+
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+/**
+ * Adds new resource adapter
+ */
+public class AddResourceAdapter implements OnlineCommand {
+    private String id = null;
+    private String module = null;
+    private String transactions = null;
```

Comment:
Fields should be `final`. Initializing them to `null` is useless, that's the default value anyway.